sgqlc/operation/__init__.py

```diff
@@ -764,11 +764,16 @@
 
 from collections import OrderedDict
 
-from ..types import ContainerType, ArgDict, global_schema
+from ..types import Union, ContainerType, ArgDict, global_schema
 
 
 DEFAULT_AUTO_SELECT_DEPTH = 2
 
+class SelectionHelpers:
+    @staticmethod
+    def supports_fields_query(type):
+        return issubclass(type, ContainerType)\
+            or issubclass(type, Union)
```

Comment:
there is no need to create a class to provide a static method. Also you can do in a single `issubclass(type, (ContainerType, Union))`.

Other than that, check the flake8, it should complain about the `type` being a reserved keyword, so use `type_` or `t` or `typ_` or something like that :-)